stacks.yml

```diff
@@ -30,10 +31,13 @@ data_storage:
   - documents_s3
   - logs_s3
   - elasticsearch
+  - monitoring
```

Comment:
Delete traverses dependencies in the opposite order. If monitoring isn't listed in the `all` group the stack won't get deleted unless explicitly mentioned